File: solutions/beecrowd/1010/1010.clj
(ns main
  (:require [clojure.string :as str]))

(defn main []
  (loop [line (read-line)
         result 0.0]
    (if (not line)
      result
      (let [[_ b c] (str/split line #" ")
            b' (Double/parseDouble b)
            c' (Double/parseDouble c)]
        (recur (read-line) (+ result (* b' c')))))))

(printf "VALOR A PAGAR: R$ %.2f%n" (main))

File: solutions/beecrowd/1010/1010.lua
local s = 0
while true do
    local line = io.read()

    if line == nil then break end

    for b, c in string.gmatch(line, "%d+ (%d+) ([0-9.]+)") do
        s = s + tonumber(b) * (tonumber(c) + 0.0)
    end
end

print(string.format('VALOR A PAGAR: R$ %.2f', s))
